### ISSUE TITLE
@module tag does not accept names with '/'

### ISSUE THREAD

**Issue by boliveira** [gajus/eslint-plugin-jsdoc#649]
## Expected behavior
@modules names should allow for / characters without any warnings, for example @module foo/bar

## Actual behavior
jsdoc/valid-types issues a warning with "Syntax error in namepath: foo/bar  jsdoc/valid-types"

## ESLint Config

```js
"extends": ["plugin:jsdoc/recommended"]
```

## ESLint sample

```js
/**
 * @module foo/bar
 */
```

## Environment

- Node version: v12.18.4
- ESLint version v6.8.0
- `eslint-plugin-jsdoc` version: 30.6.4

**Comment by boliveira** [comment]
Digging a little bit further, the error seems to be raised in jsdoctypeparser:

```
error is:  peg$SyntaxError: Expected "!", "#", ".", "...", ".<", "<", "=", "?", "[", "|", "~", [ \t], [\n], [\r], [a-zA-Z0-9_$\-], or end of input but "/" found.
    at peg$buildStructuredError (/Users/bruno.a.oliveira/Code/box/node_modules/jsdoctypeparser/peg_lib/jsdoctype-jsdoc.js:797:12)
    at peg$parse (/Users/bruno.a.oliveira/Code/box/node_modules/jsdoctypeparser/peg_lib/jsdoctype-jsdoc.js:6244:11)
    at parse (/Users/bruno.a.oliveira/Code/box/node_modules/jsdoctypeparser/lib/parsing.js:90:16)
    at validNamepathParsing (/Users/bruno.a.oliveira/Code/box/node_modules/eslint-plugin-jsdoc/dist/rules/validTypes.js:38:36)
    at /Users/bruno.a.oliveira/Code/box/node_modules/eslint-plugin-jsdoc/dist/rules/validTypes.js:180:9
    at Array.forEach (<anonymous>)
    at _default.iterateAllJsdocs (/Users/bruno.a.oliveira/Code/box/node_modules/eslint-plugin-jsdoc/dist/rules/validTypes.js:35:14)
    at iterate (/Users/bruno.a.oliveira/Code/box/node_modules/eslint-plugin-jsdoc/dist/iterateJsdoc.js:548:3)
    at /Users/bruno.a.oliveira/Code/box/node_modules/eslint-plugin-jsdoc/dist/iterateJsdoc.js:587:7
    at Array.forEach (<anonymous>)
```

**Comment by boliveira** [comment]
Confirmed it is working in 30.6.5. Thank you!
